region/maridia/outer.json

```diff
@@ -2874,6 +2874,52 @@
                   "devNote": "The global crab will fall if the speed blocks are broken."
                 }
               ]
+            },
+            {
+              "id": 6,
+              "strats": [
+                {
+                  "name": "SpeedKeep Chain",
+                  "notable": false,
+                  "requires": [
+                    {"or": [
+                      "Gravity",
+                      {"and": [
+                        "canSuitlessMaridia",
+                        "HiJump"
+                      ]}
```

Comment:
```suggestion
                      ]},
                      {"and": [
                        "canSuitlessMaridia",
                        "canSpringBallJumpMidAir"
                      ]}
```

Reply:
This is a little too hard to do because you have to be moving forward whenever you are morphed.  It'd be ok if the pause fade was instant because you can stop moving by unmorphing after the jump, but its hard to get the timings right

Reply:
I've changed my mind.  You move slow in water